way_to_home/static/src/components/map/map.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import { Map, GoogleApiWrapper } from 'google-maps-react';
+import './map.css'
+
+export class MapContainer extends Component {
+    render() {
+        return (
+            <Map
+                google={this.props.google}
+                zoom={14}
+                className='map'
+                initialCenter={{
+                    lat: 49.84,
+                    lng: 24.028667
+                }}
+            />
+        );
+    }
+}
+
+export default GoogleApiWrapper({
+
+    //Google Map Api you can find in telegram
+
+    apiKey: ''
```

Comment:
I think this is not the best approach. I found something interesting [here](https://stackoverflow.com/questions/48699820/how-do-i-hide-api-key-in-create-react-app). You may want to try it :)